docker/thehive/docker-compose.app.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+  thehive-app:
+    image: ${VAST_CONTAINER_REGISTRY:-docker.io}/tenzir/vast-poetry:${VAST_CONTAINER_REF:-latest}
+    volumes:
+      - bindings-cache:/home/vast/.cache
+      - ./app/:/opt/thehive/app # relative paths must match on host and in the container
+      - ../../python/:/python
```

Comment:
nope, `tenzir/vast-poetry` only has the VAST binary and and poetry installed. We could also install the bindings, but it would break the developer experience when working on the bindings simultaneously :-\

Reply:
Right, I now remember the discussion about this part :+1: